pkg/initiator/initiator.go

```diff
@@ -230,33 +267,36 @@ func (i *Initiator) resumeLinearDmDevice() error {
 func (i *Initiator) replaceDmDeviceTarget() error {
 	suspended, err := i.IsSuspended()
 	if err != nil {
-		return errors.Wrapf(err, "failed to check if linear dm device is suspended for NVMe-oF initiator %s", i.Name)
+		return errors.Wrapf(err, "failed to check if linear dm device is suspended for initiator %s", i.Name)
 	}
 
 	if !suspended {
 		if err := i.suspendLinearDmDevice(true, false); err != nil {
-			return errors.Wrapf(err, "failed to suspend linear dm device for NVMe-oF initiator %s", i.Name)
+			return errors.Wrapf(err, "failed to suspend linear dm device for initiator %s", i.Name)
 		}
 	}
 
 	if err := i.reloadLinearDmDevice(); err != nil {
-		return errors.Wrapf(err, "failed to reload linear dm device for NVMe-oF initiator %s", i.Name)
+		return errors.Wrapf(err, "failed to reload linear dm device for initiator %s", i.Name)
 	}
 
 	if err := i.resumeLinearDmDevice(); err != nil {
-		return errors.Wrapf(err, "failed to resume linear dm device for NVMe-oF initiator %s", i.Name)
+		return errors.Wrapf(err, "failed to resume linear dm device for initiator %s", i.Name)
 	}
 	return nil
 }
 
-// Start starts the NVMe-oF initiator with the given transportAddress and transportServiceID
-func (i *Initiator) Start(transportAddress, transportServiceID string, dmDeviceAndEndpointCleanupRequired bool) (dmDeviceIsBusy bool, err error) {
+// StartNvmeTcpInitiator starts the NVMe-oF initiator with the given transportAddress and transportServiceID
+func (i *Initiator) StartNvmeTcpInitiator(transportAddress, transportServiceID string, dmDeviceAndEndpointCleanupRequired bool) (dmDeviceIsBusy bool, err error) {
```

Comment:
For naming alignment
```suggestion
func (i *Initiator) StartNvmeTCPInitiator(transportAddress, transportServiceID string, dmDeviceAndEndpointCleanupRequired bool) (dmDeviceIsBusy bool, err error) {
```

Reply:
NIT: I am thinking if we can put `transportAddress, transportServiceID` in `nvmeTcpInfo` during `NewInitiator()`. Then we can use a single general function `func (i *Initiator) Start(dmDeviceAndEndpointCleanupRequired bool) (dmDeviceIsBusy bool, err error)` here

Reply:
> For naming alignment

Thanks @derekbit Fixed

Reply:
> NIT: I am thinking if we can put `transportAddress, transportServiceID` in `nvmeTcpInfo` during `NewInitiator()`. Then we can use a single general function `func (i *Initiator) Start(dmDeviceAndEndpointCleanupRequired bool) (dmDeviceIsBusy bool, err error)` here

@shuo-wu 
I think we should not put  `transportAddress, transportServiceID` in `nvmeTcpInfo` during `NewInitiator()` because we later need to verify it here https://github.com/longhorn/go-spdk-helper/blob/80f4fdf02d3958dcde2d9683ab0460a5f8e2f4d4/pkg/initiator/initiator.go#L319-L331. WDYT?

Reply:
OK. Then let's retain the current implementation